src/dimension.c

```diff
@@ -564,6 +565,49 @@ dimension_scan_internal(ScanKeyData *scankey, int nkeys, tuple_found_func tuple_
 	return ts_scanner_scan(&scanctx);
 }
 
+/*
+ * Check if there are any correlated constraint entries and if yes create
+ * two new structures in the hypertable
+ */
+void
+ts_correlated_constraints_assign(Hypertable *h, MemoryContext mctx)
+{
+	int i;
+	int num_corr_cons = 0;
+	Hyperspace *hs = h->space, *sec_hs = NULL, *new_hs = NULL;
+
+	for (i = 0; i < hs->num_dimensions; i++)
+	{
+		Dimension *dim = &hs->dimensions[i];
+
+		if (IS_CORRELATED_CONSTRAINT(dim))
```

Comment:
I guess this makes more sense?

```suggestion
		if (IS_CORRELATED_DIMENSION(dim))
```

Reply:
Ok, done.